app/page.tsx

```diff
@@ -31,12 +32,15 @@ const isLookerEnabled = process.env.IS_LOOKER_ENABLED === "true";
 const hidden_message_impression_threshold =
   process.env.HIDDEN_MESSAGE_IMPRESSION_THRESHOLD;
 
-function compareFn(a: any, b: any) {
-  if (a._rawRecipe.startDate > b._rawRecipe.startDate) {
-    return -1;
-  } else if (a._rawRecipe.startDate < b._rawRecipe.startDate) {
-    return 1;
+function compareDatesFn(a: NimbusRecipe, b: NimbusRecipe): number {
```

Comment:
Please add a JSDoc comment here to make it explicit to the reader that if one or both of the recipes is missing a start date, they will be ordered identically when using this compare function. Describing why this is desirable behavior seems worthwhile too, even if it's merely an arbitrary choice, since this will help future changes consider this.